RedHat-UX/red-hat-design-system
# perf(footer): allow footer templates to be minified

Closes #558 

## What I did

1. remove `unsafeStatic` from footer templates


## Testing Instructions

1. check out this branch
```
npm ci
run build
cd /tmp 
mkdir rollup-repro-lit-templates
cd rollup-repro-lit-templates
npm init -y
npm i @rhds/elements @rollup/plugin-node-resolve rollup-plugin-minify-html-literals
touch rollup.config.js
echo "export * from '@rhds/elements/rh-footer/rh-footer.js'" > input.js
```

Then *copy* the build files from this branch. season pathnames to taste

```
cp ~/Developer/redhat-ux/red-hat-design-system/elements/rh-footer/*.js node_modules/@rhds/elements/elements/rh-footer/
```

Use this rollup config
```js
import minifyTemplateLiterals from 'rollup-plugin-minify-html-literals'
import nodeResolve from '@rollup/plugin-node-resolve'
export default {
  input: 'index.js',
  output: {
    file: 'output.js'
  },
  plugins: [
    nodeResolve(),
    minifyTemplateLiterals(),
  ]
}
```

then do
```
rollup -c
``` 
and you should not see any warnings related to html template minifier. the output.js should have minified templates.

## Notes to Reviewers
